.eslintrc.js

```diff
@@ -1,45 +1,27 @@
 module.exports = {
-  extends: [
-    '@metamask/eslint-config',
-    '@metamask/eslint-config/config/nodejs',
-  ],
-  plugins: [
-    'json',
-  ],
-  parserOptions: {
-    ecmaVersion: 2018,
-  },
+  root: true,
+
+  extends: ['@metamask/eslint-config'],
+
   overrides: [
     {
       files: ['*.ts'],
-      extends: [
-        '@metamask/eslint-config/config/typescript',
-      ],
+      extends: ['@metamask/eslint-config-typescript'],
     },
+
     {
-      files: [
-        '*.js',
-        '*.json',
-      ],
+      files: ['*.js'],
       parserOptions: {
         sourceType: 'script',
       },
-      rules: {
-        '@typescript-eslint/no-require-imports': 'off',
-        '@typescript-eslint/no-var-requires': 'off',
-      },
+      extends: ['@metamask/eslint-config-nodejs'],
     },
+
     {
-      files: ['test/*'],
-      rules: {
-        // This happens before a build is created. It doesn't matter.
-        'import/no-unresolved': 'off',
-      },
+      files: ['*.json'],
+      extends: ['plugin:json/recommended'],
```

Comment:
We have dropped the JSON plugin elsewhere in favor of prettier. The ESLint JSON plugin is not great, it's capable of doing very little. Prettier does everything this plugin does and more.

Reply:
Ah yes, you're right. I did end up replacing this with Prettier in #97, but happy to do that here if it makes more sense.

Reply:
Ah I see - I didn't realize that `prettier` wasn't being invoked directly in this PR yet. Handling it in the next PR seems fine to me.